src/components/AsideHeader/useAsideHeaderInnerContextValue.tsx

```diff
@@ -53,6 +64,25 @@ export const useAsideHeaderInnerContextValue = (
         [innerOnClosePanel, ALL_PAGES_MENU_ITEM, onClosePanel],
     );
 
+    const onToggleMenuGroupVisibility = useCallback(
```

Comment:
The toggle group visibility logic exists in two places:
- AllPagesPanel.tsx:98-115 (`toggleGroupVisibility`)
- useAsideHeaderInnerContextValue.tsx:67-84 (`onToggleMenuGroupVisibility`)

They do similar but different things - one toggles hidden, the other toggles collapsed.

Recommendation: Rename these functions to be clearer about their purpose:
`toggleGroupVisibility` → `toggleGroupHidden`
`onToggleMenuGroupVisibility` → `onToggleGroupCollapsed`